components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -17,6 +17,18 @@ import java.nio.ByteOrder
 import java.util.concurrent.atomic.AtomicLong
 import java.util.concurrent.atomic.AtomicReference
 import java.lang.ref.WeakReference
+import org.mozilla.appservices.places.GleanMetrics.PlacesManager as PlacesManagerMetrics
+
+/**
+ * Import some private Glean types, so that we can use them in type declarations.
+ *
+ * By agreement with the Glean team, we must not
```

Comment:
> I wonder if we should escalate this somehow so that we can do what we need to do
> without these scary caveats?

It's on file as an enhancement request here: https://bugzilla.mozilla.org/show_bug.cgi?id=1601553

Reply:
(Also FWIW the conversation about it with the Glean team was quite amicable 😄)